src/main/kotlin/org/openrewrite/RecipeMarkdownGenerator.kt

```diff
@@ -212,6 +212,14 @@ class RecipeMarkdownGenerator : Runnable {
         val markdownArtifacts = TreeMap<String, MarkdownRecipeArtifact>()
         val recipesWithDataTables = ArrayList<RecipeDescriptor>()
         val moderneProprietaryRecipes = TreeMap<String, MutableList<RecipeDescriptor>>()
+        
+        // Build reverse mapping of recipe relationships (which recipes contain each recipe)
+        val recipeContainedBy = mutableMapOf<String, MutableList<RecipeDescriptor>>()
```

Comment:
We can likely use this same mapping when it comes to recipes not included anywhere.